ansible/roles/software/dashboard/dashboard-deploy/templates/.env

```diff
@@ -2,7 +2,17 @@ DB_HOST='{{ env_db_host }}'
 DB_NAME='{{ env_db_name }}'
 DB_PASSWORD='{{ env_db_pass }}'
 DB_USER='{{ env_db_user }}'
+# DB_DEBUG:
+#   TRUE: fail on database errors on migration, display to user
```

Comment:
:sob: WTF code igniter!

Reply:
Though it's a built-in config option for CodeIgniter's database class, we added the option to set it via the `.env`. It's set to default to FALSE, at least. 

Actually I just realized it's no longer used by the code, though... See [this line](https://github.com/GSA/project-open-data-dashboard/blob/ea376e75ac09770b4de979d4e05a7a75818fce2e/application/config/database.php#L92) carried in from the upstream CodeIgniter base. I like that default logic better; will yank this option out of `.env` entirely in another commit.